src/platforms/javascript/common/configuration/install-cdn.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Installing with CDN
+caseStyle: camelCase
+supportLevel: production
+categories:
+  - browser
+---
+On this page, we provide instructions to install Sentry's JavaScript SDK using content delivery network (CDN), automatically reporting errors and exceptions in your application.
```

Comment:
```suggestion
On this page, we provide instructions to install Sentry's JavaScript SDK using a content delivery network (CDN), so that it will automatically report errors and exceptions in your application.
```